go/firewallsso/fortigate.go

```diff
@@ -59,13 +63,13 @@ func (fw *FortiGate) Stop(ctx context.Context, info map[string]string) (bool, er
 // Build the RADIUS packet for an SSO stop
 func (fw *FortiGate) stopRadiusPacket(ctx context.Context, info map[string]string) *radius.Packet {
 	r := radius.New(radius.CodeAccountingRequest, []byte(fw.Password))
-	r.Set("Acct-Session-Id", "acct_pf-"+info["mac"])
-	r.Set("Acct-Status-Type", uint32(2))
-	r.Set("User-Name", info["username"])
-	r.Set("Class", info["role"])
-	r.Set("Called-Station-Id", "00:11:22:33:44:55")
-	r.Set("Framed-IP-Address", net.ParseIP(info["ip"]))
-	r.Set("Calling-Station-Id", info["mac"])
+	rfc2866.AcctSessionID_AddString(r, "acct_pf-"+info["mac"])
+	rfc2866.AcctStatusType_Add(r, rfc2866.AcctStatusType(2))
```

Comment:
constant

Reply:
use rfc2866.AcctStatusType_Value_Stop instead of rfc2866.AcctStatusType(2)